lib/utils/formatter.go

```diff
@@ -326,7 +326,7 @@ func (w *writer) writeMap(m map[string]interface{}) {
 			continue
 		}
 		switch value := m[key].(type) {
-		case log.Fields:
+		case map[string]interface{}:
```

Comment:
This will let a `log.Fields` through to `writeKeyValue`, won't it? Should we use `writeMap` for both?

Also, what's the logic behind not actually printing the key for a map value? 🤔

Reply:
No it won't because the function signature is `writeMap(map[string]interface{}) so even though it accepts log.Fields the type will be a plain map.

As for other that's a good question but not sure it would be a good idea to change it at this point. 🤷

Reply:
Yes, but currently a `map[string]any` that contains a `log.Fields` in a value will print out the value with `writeMap` (not sure how many of those are used as _values_, but still) whereas a `map[string]any` will get passed to `writeKeyValue`. Are we ok with potentially moving this flaw from a `map[string]any` value into a `log.Fields` value?

Reply:
Sorry, you are correct. I had looked through the code and verified that nobody was using log.Fields as the value in a map so I discounted that but upon reflection I think you are correct that it is better to be safe. I've pushed up the changes